content/en/blog/wireshark.md

```diff
@@ -119,52 +118,108 @@ service PersonSearchService {
+[grpc_person_search_protobuf_with_image.pcapng][].
+
+<!--
+TODO: Why port 50052? I'm not seeing any traffic over port 50052 in the
+      screenshots. Is it actually used?
+-->
+The app's gRPC traffic was captured on ports 50051 and 50052. Before you can
+decode these messages, first register these ports as carrying HTTP2 traffic:
+
+<!--
+TODO: Clarify: how does the user access this configuration pane?
+TODO: Is the port 18127 relevant to this blog post, if not, can you remove it?
+TODO: Is the path "C:\Users\...decode_as_entries" actually part of the dialog? If not, can you remove it?
+-->
 ![wireshark_decode_as_dialog](/img/wireshark_decode_as_dialog.png)
 
-This is a screenshot of the gRPC service request message of
-[grpc_person_search_protobuf_with_image.pcapng](https://gitlab.com/wireshark/wireshark/-/wikis/uploads/f6fcdceb0248669c0b057bd15d45ab6f/grpc_person_search_protobuf_with_image.pcapng):
+### Decoding the search request message
+
+<!--
+TODO: What are the steps that the user needs to follow before the user can see
+      the decoded message as shown in the screenshot?
+-->
+Here's a screenshot of the sample's service request message:
+
+<!--
+TODO: Can you remove the commentary text (in blue) from the image. Such text is
+      best placed in the body of the post (it makes it easier to edit, and
+      eventually possibly translate into other languages :))
+
+TODO: If a users opens up the "GRPC_Message" section in that display, what will
+      be shown?
+-->
 ![wireshark_grpc_protobuf_search_request](/img/wireshark_grpc_protobuf_search_request.png)
-That shows searching for the Person objects of Jason and Lily based on their
-names.
+
+By examining the HTTP2 message header `path` field, you'll notice the URL to the
+app's service (`/tutorial.PersonSearchService`), followed by the name of the
+invoked RPC (`Search`).
+
+The `content-type`, which is set by the gRPC library, informs Wireshark that the
+HTTP2 message content is a gRPC message. By examining the decoded Protocol
+Buffers message of the sample gRPC request, you can see that the search request
+is for the names "Jason" and "Lily".
+
+### Decoding the server-streamed response
 
 Since the `Search` operation is defined as the server streaming RPC mode, the
 Person objects can be return back to client one after another:
+
+<!--
+
+TODO: The response is sent over port 51035. Does a user need to configure the
+      "decoding as" HTTP2 for this port?
+
+      It might also be worth mentioning to the reader how we know that 51035 is the response port.
+-->
 ![wireshark_grpc_protobuf_search_response](/img/wireshark_grpc_protobuf_search_response.png)
 
 You may have noticed that the `portrait_image` of bytes type is parsed as a
-PNG data. It's because we register the PNG dissector in the `"protobuf_field"`
+PNG data.
+<!--
+TODO: Question: does the user actually see the image? If so, that isn't obvious from the screenshot.
```

Comment:
User could not see the image directly, instead it contains some data about the image like:
```
portrait_image: (119 bytes)
    Portable Network Graphics
        PNG Signature: 89504e470d0a1a0a
        Image Header (IHDR)
        Standard RGB colour space (sRGB)
        Image gamma (gAMA)
        Physical pixel dimensions (pHYs)
        Image data chunk (IDAT)
        Image Trailer (IEND)
```
But user can export the binary data of the image to a file (for example named lily.png) by right clicking the node and selecting the 'export packet bytes..' menu item, and open the image with system picture viewer.

Reply:
Given the next comment, this won't be relevant anymore.